public/models/interfaces.ts

```diff
@@ -184,7 +197,20 @@ export type AnomalySummary = {
   lastAnomalyOccurrence: string;
 };
 
+export type HistoricalDetectorAnomalySummary = {
```

Comment:
Is this being used? Looks like [AnomalySummary](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/14168a737f361203d0b2434006b04d55c65c0dfd/public/models/interfaces.ts#L178) can be reused.

Reply:
This is a good catch. And actually, the reason for having a separate data model here is because the `lastAnomalyOccurrence` field is not applicable for the historical detector case, and is not part of the mockups.

Looks like currently, the `HistoricalDetectorAnomalySummary` type is not being used properly in the `AnomalyHistory` component. I will need to add a check to show the appropriate summary based on the detector type.

Reply:
After looking more, seems it will be easier to not add a new `HistoricalDetectorAnomalySummary` here at all like you suggested, and instead just remove the occurrence item if render historical detector results. Will remove  `HistoricalDetectorAnomalySummary`.

Reply:
Per the mockup, have removed the confidence showing on the chart entirely for the historical detector case.